modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheAdapter.java

```diff
@@ -2110,6 +2110,12 @@ protected boolean put0(final K key, final V val, final CacheEntryPredicate filte
         if (F.isEmpty(drMap))
             return;
 
+        if (log.isDebugEnabled()) {
+            for (KeyCacheObject keyCacheObject : drMap.keySet()) {
+                log.debug("PutAllConflict invoked for the key = " + keyCacheObject.value(ctx.cacheObjectContext(), false));
+            }
+        }
+
```

Comment:
Let's move this to IgniteDrDataStreamerCacheUpdater.
Let's log at trace log level.